packages/core/src/fields/types/relationship/views/cards/index.tsx

```diff
@@ -154,6 +154,9 @@ export function Cards({
       >
         {[...value.currentIds].map((id, index) => {
           const itemGetter = items[id];
+          if (!itemGetter) {
+            return null;
+          }
```

Comment:
Could we improve the type of `items` to lint for this?

Reply:
Not practically - could technically make the type `Record<string, Blah | undefined>` but that would be conceptually wrong and mean having to do a non-null assertion in places or we could use `noUncheckedIndexedAccess` but that would mean _a lot_ of errors and a lot of non-null assertions